syntax = "proto3";

import "findit.proto";

package findit;

// Schema for a test analysis event.
message TestAnalysisCompletionEvent {
  // Base analysis info.
  Analysis analysis_info = 1;

  // Name of the test.
  string test_name = 10;

  // Test type (flake or failure).
  bool flake = 11;
}